Sources/StringsLintFramework/Rules/Configuration/StructuredPlaceholderCommentConfiguration.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  StructuredPlaceholderCommentConfiguration.swift
+//  StringsLintFramework
+//
+//  Created by Mark Hall on 2022-07-18.
+//
+
+import Foundation
+
+public struct StructuredPlaceholderCommentRuleConfiguration: RuleConfiguration {
+    public var description: String {
+        return "severity: \(self.severity)"
+    }
+
+    //TODO: (Mark Hall, July 18) once we have all the existing strings converted to the new comment format, make this an error severity
+    public var severity: ViolationSeverity = .warning
```

Comment:
what will be our strategy to migrate all. We need to communicate to the broader iOS team before we merge the lint rule

Reply:
I think we should enable this warning for all strings, and then separate the `.strings` and `.stringsdict` files out amongst the iOS devs and get them all to be accountable for updating the strings and removing the warnings. This is what you outlined in the DRE doc. 
Once all strings are done, we'll convert this to a `.error`

Reply:
I agree. We just need to make sure the outlined plan moves forward. You would be responsible for ensuring it does :)